src/components/transactions/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Waypoint from 'react-waypoint';
+import tableStyle from 'react-toolbox/lib/table/theme.css';
+import { transactions } from '../../utils/api/account';
+import TransactionsHeader from './transactionsHeader';
+import TransactionRow from './transactionRow';
+
+class Transactions extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      transactions: [],
+      offset: 0,
+      loadMore: true,
+      length: 1,
+    };
+  }
+
+  componentDidMount() {
+    this.loadMore();
+  }
+
+  loadMore() {
+    if (this.state.loadMore && this.state.length > this.state.offset) {
+      this.setState({ loadMore: false });
+      transactions(this.props.activePeer, this.props.address, 20, this.state.offset)
+      .then((res) => {
+        const list = res.transactions.map(transaction => (
+          <TransactionRow address={this.props.address}
```

Comment:
I think that the state should contain the list of transactions returned from the server, not list of React components.